bin/wagon

```diff
@@ -350,6 +363,11 @@ complete -F __wagon_completion -o default wagon
 COMPLETION
     ;;
 
+  # TODO: add these in the right place
```

Comment:
Brauchts das TODO noch? Soweit ich das sehe sind die jetzt alle da

Reply:
In der Completion sindn sie vorhanden, aber leider nicht in der USAGE-Hilfe. Daher ist nicht ganz klar, ob man einen Wagon angeben muss oder nicht. Merkt man recht schnell, aber halt auf die harte Tour.